docker/docker-compose/docker-compose.yml

```diff
@@ -123,4 +136,20 @@ services:
     ports:
       - "8008:8008"
     volumes:
-      - ./collect-data:/data/collect-data
\ No newline at end of file
+      - ./collect-data:/data/collect-data
+
+  audit:
+    image: inlong/audit:latest
+    container_name: audit
+    depends_on:
+      mysql:
+        condition: service_healthy
```

Comment:
@dockerzhang BTW, version 3 of Docker Compose no longer supports the `condition` form of `depends_on`, should we adapt it?